clappr/clappr-ios
# Change core visibility in player

Goal
===
Change the visibility of the Player's core.

How to test
===
1. Run iOS and tvOS unit tests.
2. Run the sample app and check if everything works as expected.
